checker/src/org/checkerframework/checker/index/lowerbound/LowerBoundAnnotatedTypeFactory.java

```diff
@@ -95,6 +95,42 @@ public LowerBoundAnnotatedTypeFactory(BaseTypeChecker checker) {
                         LowerBoundUnknown.class));
     }
 
+    /**
+     * Takes a value type (only interesting if it's an IntVal), and converts it to a lower bound
+     * type. If the new lower bound type is more specific than type, convert type to that type.
+     *
+     * @param valueType the value checker's type
+     * @param type the current lower bound type of the expression being evaluated
+     */
+    private void addLowerBoundTypeFromValueType(
+            AnnotatedTypeMirror valueType, AnnotatedTypeMirror type) {
+        AnnotationMirror anm = getLowerBoundAnnotationFromValueType(valueType);
+        if (!type.isAnnotatedInHierarchy(UNKNOWN)) {
+            type.addAnnotation(anm);
```

Comment:
Type variables may or may not have a "primary" annotation -- that is AnnotatedTypeVariable#getAnnotations() may be empty, even if the type is well formed and completely annotated.  In this case, this branch is taken.  If 'anm' is `@LowerBoundUnknown`, it is added as a primary annotation, which sets the annotation on the upper and lower bound to `@LowerBoundUnknown`, too. (This cause the error you reported.)

So, to fix the error, don't add 'anm' if it is `@LowerBoundUnknown`.